lite-uda-installer-archives-mapping.ttl

```diff
@@ -10112,6 +10112,41 @@ source:
     oplinst:isFileSizeSpecificationOf <http://data.openlinksw.com/oplweb/installer/UDASTLitehttps3m1l9g7zzdmg90macOS11Universal#this> ;
     wdrs:describedby source: .
 
+<http://data.openlinksw.com/oplweb/installer/UDASTLitehttps3m1jczzzzdmg90macS11Universal#this>
+    a oplinst:InstallerArchive ,
+        schema:CreativeWork ,
+        oplinst:UDASTInstallerArchive ,
+        oplinst:UDASTLiteInstallerArchive ,
+        oplinst:UDASTLite80InstallerArchive ;
+    dcterms:format "application/x-apple-diskimage" ;
+    dcterms:identifier "m1jczzzz.dmg" ;
+    schema:dateCreated "2025-05-12T00:00:00-05:00"^^xsd:dateTime ;
+;    schema:dateModified "2021-07-05T20:46:00-00:00"^^xsd:dateTime ;
```

Comment:
How was it modified 4 years before it was created?

Reply:
Clearly a typo!